### PR TITLE
Add basic ONBOARDING doc with basic getting started information

### LINKED ISSUE #101: onboarding documentation
Task from todays call to produce the onboarding documentation

### PR DESCRIPTION
**Pull request by hspinks**
Very basic initial onboarding pointing to resources for new folks to get started.

Closes #101

### CODE REVIEW

**Comment by hspinks** [comment]
Pretty basic to start, can continue building this out as we go

File: ONBOARDING.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+* Most up-to-date view of app designs: TBD
+* Most up-to-date preview of app in development: TBD
+
+Most discussion will be taking place in a combination of Slack and Github issues, so those are your best places to start.
+
+## Point People
+
+This is very much a committee effort, and we are moving quickly enough that we have to self-organize as much as we can. You're encouraged to step up and find places to contribute rather than relying on anyone to tell you what to do. Even in just the first couple days of the project, there has been incredible effort from a lot of contributors. With that said, there are a couple people helping play point around different areas that are trying to keep a pulse of everything going on in those areas:
+
+* Karen Wong - all things Product
```
Comment (advayDev1) [inline ONBOARDING.md]: add @ github usernames for all three

File: ONBOARDING.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
+# Getting Started
+
+If you're here, that means you're interested in being a part of the volunteer effort. Welcome! We're glad to have you.
+
+The goal of this document is to give you a starting point for where to find the resources you need to get up to speed. This is by no means an exhaustive document, and I'm sure that we'll be missing some important items here, so if you're missing something, just ask and we will try to update this document.
+
+First thing's first - in order to keep track of everyone working on the project, please add yourself to the project roster: https://docs.google.com/spreadsheets/d/1jt4xrXWUnsh8DmtawNV4XGBj8RAdgn-d7ckgL2qEXPM/edit#gid=0
+
+Next - introduce yourself in the #intros channel on [Slack](https://join.slack.com/t/who-app/shared_invite/zt-cylowls6-Q7iQPvTAenkN_Yb9MLxVIw)!
```
Comment (advayDev1) [inline ONBOARDING.md]: See here: https://who-app.slack.com/archives/C010C6XJ80N/p1584740224130300

File: ONBOARDING.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+* Karen Wong - all things Product
+* Vivian Cromwell - all things Design
+* Hunter Spinks - all things Engineering
+
+# Getting Started
+
+Please review the [Contributor Guidelines](https://github.com/WorldHealthOrganization/app/blob/master/CONTRIBUTING.md) before making any contributions to the code. Note that you'll need to add yourself to the [LICENSE](LICENSE) contributor list on your first commit.
+
+This app uses a monorepo structure with the client application in the `client` directory and all server-related code in the `server` directory.
```
Comment (advayDev1) [inline ONBOARDING.md]: "(currently only the client/ionic directory is used)"

**Comment by advayDev1** [comment]
(process: think we should "assign" PRs to the person, if any, whose review is blocking submission)

File: README.md [hunk 4]
```diff
@@ -30,6 +30,10 @@ We want to get this out and working so we can move on to more significant functi
 
 We are assessing the feature roadmap (contact tracing and more) for the next and subsequent releases.
 
+## Getting Started
+
+See [ONBOARDING](https://github.com/WorldHealthOrganization/app/blob/master/ONBOARDING.md)
```
Comment (epicfaace) [inline README.md]: I'd suggest renaming this to CONTRIBUTING.md -- it's a more standard convention for open source repos. See https://help.github.com/en/github/building-a-strong-community/setting-guidelines-for-repository-contributors#adding-a-contributing-file

File: ONBOARDING.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+Where to find answers to your questions:
+
+* Github Issues: https://github.com/WorldHealthOrganization/app/issues
+* Slack: https://join.slack.com/t/who-app/shared_invite/zt-cylowls6-Q7iQPvTAenkN_Yb9MLxVIw
+* Initial project brief (will be updated): https://docs.google.com/document/d/1isNMLpwI2iUY92KPwJHfY7kQnpN3oCuUl6c94J7Qmhs
+* Rough engineering backlog: https://github.com/WorldHealthOrganization/app/projects/1
+* Most up-to-date view of app designs: TBD
+* Most up-to-date preview of app in development: TBD
+
+Most discussion will be taking place in a combination of Slack and Github issues, so those are your best places to start.
```
Comment (epicfaace) [inline ONBOARDING.md]:
```suggestion
Most discussion will be taking place in a combination of Slack and GitHub issues, so those are your best places to start.
```

File: ONBOARDING.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+
+First thing's first - in order to keep track of everyone working on the project, please add yourself to the project roster: https://docs.google.com/spreadsheets/d/1jt4xrXWUnsh8DmtawNV4XGBj8RAdgn-d7ckgL2qEXPM/edit#gid=0
+
+Next - introduce yourself in the #intros channel on [Slack](https://join.slack.com/t/who-app/shared_invite/zt-cylowls6-Q7iQPvTAenkN_Yb9MLxVIw)!
+
+## Resources
+
+Where to find answers to your questions:
+
+* Github Issues: https://github.com/WorldHealthOrganization/app/issues
```
Comment (epicfaace) [inline ONBOARDING.md]:
```suggestion
* GitHub Issues: https://github.com/WorldHealthOrganization/app/issues
```

File: README.md (hunk 4 above)
Comment (epicfaace) [reply README.md]: oops, never mind, we already have a CONTRIBUTING.md -- please disregard my comment!

Comment (hspinks) [reply README.md]: definitely a fair comment! I think the idea behind this is that so many people are coming in from all over on this project, so this is meant to be a bit more organizational information beyond just the contributing guidelines. maybe we eventually merge the two 👍

File: ONBOARDING.md (hunk 6 above)
Comment (hspinks) [reply ONBOARDING.md]: fixed in https://github.com/WorldHealthOrganization/app/pull/104/commits/1c3f2b4d9fd88609d8a764d7e6c5b8880e374b6c

File: ONBOARDING.md (hunk 5 above)
Comment (hspinks) [reply ONBOARDING.md]: fixed in https://github.com/WorldHealthOrganization/app/pull/104/commits/1c3f2b4d9fd88609d8a764d7e6c5b8880e374b6c

File: ONBOARDING.md (hunk 3 above)
Comment (hspinks) [reply ONBOARDING.md]: updated in https://github.com/WorldHealthOrganization/app/pull/104/commits/1c3f2b4d9fd88609d8a764d7e6c5b8880e374b6c

File: ONBOARDING.md (hunk 2 above)
Comment (hspinks) [reply ONBOARDING.md]: added in https://github.com/WorldHealthOrganization/app/pull/104/commits/1c3f2b4d9fd88609d8a764d7e6c5b8880e374b6c

File: ONBOARDING.md (hunk 1 above)
Comment (hspinks) [reply ONBOARDING.md]: added in https://github.com/WorldHealthOrganization/app/pull/104/commits/1c3f2b4d9fd88609d8a764d7e6c5b8880e374b6c

File: ONBOARDING.md [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+ionic capacitor copy android
+ionic capacitor run android -l --host=YOUR_IP_ADDRESS
+```
+
+To run on an iPhone Simulator using Xcode, from the `client/ionic` directory run:
+
+```
+ionic build
+ionic capacitor copy ios
+ionic capacitor open ios
```
Comment (SamMousa) [inline ONBOARDING.md]: Is this correct? I'd assume android and ios use similar commands.

Comment (hspinks) [reply ONBOARDING.md]: this is slightly different than what I've listed for the Android version... this command opens Xcode to run, giving you access to the full project settings which may be useful as we get closer to deployment, have different environments, etc. iOS has a similar command as Android to run `ionic capacitor run ios -l --external`, but I'm personally a little more comfortable working with all of the settings in Xcode. I'll document that difference...

Comment (SamMousa) [reply ONBOARDING.md]: https://ionicframework.com/docs/cli/commands/capacitor-run

It should do both.

**Comment by SamMousa** [comment]
Lgtm!

**Pull request merged**